frontend/src/metabase/ui/components/overlays/Modal/Modal.styled.tsx

```diff
@@ -0,0 +1,36 @@
+import { t } from "ttag";
+import type { MantineThemeOverride } from "@mantine/core";
+
+export const getModalOverrides = (): MantineThemeOverride["components"] => ({
+  Modal: {
+    styles: theme => ({
+      title: {
+        fontSize: theme.fontSizes.xl,
+        fontWeight: 700,
+      },
+      close: {
+        color: theme.colors.text[0],
+        ":hover": {
+          color: theme.colors.text[1],
+        },
+      },
+      overlay: {
+        backgroundColor: theme.fn.rgba(theme.colors.bg[3], 0.6),
+      },
```

Comment:
Follows [the current backdrop color](https://github.com/metabase/metabase/blob/45315fa445f087cbc216d6e935555028e28032d8/frontend/src/metabase/css/components/modal.css#L58)